src/main/java/io/jenkins/plugins/analysis/core/util/ModelValidation.java

```diff
@@ -31,6 +31,8 @@
 
     @VisibleForTesting
     static final String NO_REFERENCE_JOB = "-";
+    @VisibleForTesting
```

Comment:
This file needs no modifications (since not meaningful for freestyle jobs)

Reply:
Fixed.